src/config/network-filesystems.md

```diff
@@ -0,0 +1,72 @@
+# Network Filesystems
+
+## NFS
+
+### Mounting an NFS Share
+
+To mount an NFS share, start by installing the `nfs-utils` and `sv-netmount`
+packages.
+
+Before mounting an NFS share, [enable](./services/index.md#enabling-services)
+the `statd`, `procbind`, and `sv-netmount` services. If the server supports
+`nfs4`, the `statd` service isn't necessary.
+
+To mount an NFS share:
+
+```
+# mount -t <mount_type> <host>:/path/to/sourcedir /path/to/destdir
+```
+
+`<mount_type>` should be `nfs4` if the server supports it, or `nfs` otherwise.
+`<host>` can be either the hostname or IP address of the server.
+
+Mounting options can be found in
+[mount.nfs(8)](https://man.voidlinux.org/mount.nfs.8), while unmounting options
+can be found in [umount.nfs(8)](https://man.voidlinux.org/umount.nfs.8).
+
+For example, to connect `/volume` on a server at `192.168.1.99` to a directory
+`/mnt/volume` on your local system:
+
+```
+# mkdir /mnt/volume
+# mount -t nfs 192.168.1.99:/volume /mnt/volume
+```
+
+To have the directory mounted when the system boots, add an entry to
+[fstab(5)](https://man.voidlinux.org/fstab.5):
+
+```
+192.168.1.99:/volume /mnt/volume nfs rw,hard,intr 0 0
+```
+
+Refer to [nfs(5)](https://man.voidlinux.org/nfs.5) for more information about
+the available options.
```

Comment:
"for information about the avaiable mounting options" ?

Reply:
Done.